fcrepo/fcrepo
# Fcrepo 3839

**Add Banner with registration pitch and link to registration page to All HTML UI pages**
* * *

**JIRA Ticket**: https://fedora-repository.atlassian.net/browse/FCREPO-3839

* Part of https://fedora-repository.atlassian.net/browse/FCREPO-3840

# What does this Pull Request do?
Adds a HTML UI banner with text and a link to the repository registration site.

# How should this be tested?
View in a variety of common browsers and confirm the new configuration property ```fcrepo.banner.display=on/off``` is working as expected. THe UI feature should be visible at the top of all pages within the UI other than the entry 'splash screen' (as found at /fcrepo)


# Interested parties
@fcrepo/committers
